code/counter-app/generate-demos.sh

```diff
@@ -0,0 +1,6 @@
+#/bin/bash
+
+for i in *.tape; do
+    ~/go/bin/vhs $i
+done
+rm -f basic-app-*.gif
```

Comment:
I used `brew` to install `vhs`. 

```
$ which vhs
/opt/homebrew/bin/vhs
```

Is there a reason this is `~/go/bin/vhs` instead of `vhs`?

Reply:
Yep - the Screenshot command and theme updates haven't been released yet. VHS seems to lag on releases for many months, so installing using go install is the right approach. I noted that in a bunch of the tape files, but it makes sense to mention it here too.

Reply:
homebrew has v0.7.1 now (which has the updated themes and screenshot command)